symbolic-debuginfo/src/sourcebundle.rs

```diff
@@ -1035,6 +1036,35 @@ fn sanitize_bundle_path(path: &str) -> String {
     sanitized
 }
 
+/// Contains information about a file skipped in the SourceBundleWriter
+#[derive(Debug)]
+pub struct SkippedFileInfo<'a> {
+    path: &'a str,
+    reason: &'a str,
+}
+
+impl<'a> SkippedFileInfo<'a> {
+    fn new(path: &'a str, reason: &'a str) -> Self {
+        Self { path, reason }
+    }
+
+    /// Get the path of the skipped file
```

Comment:
_nit_: Suggestion based on the Rust style guidelines. Similar for other methods. Also note the punctuation.

```suggestion
    /// Returns the path of the skipped file.
```